src/pages/gen2/build-a-backend/data/relationships/index.mdx

```diff
@@ -12,3 +12,298 @@ export function getStaticProps(context) {
+    comments: a.hasMany('Comment'),
+  }),
+  Comment: a.model({
+    content: a.string(),
+    post: a.belongsTo('Post'),
+  }),
+});
+
+const client = generateClient<Schema>();
+
+const { data: post } = await client.models.Post.get('FABF27EE-25D2-4851-AE1C-09BCA1604C77');
+
+(await post.comments())[0].post === post; // Navigating back to the post from the comment
+```
+
+### Many-to-many relationship
+
+Create a many-to-many relationship between two models with the `manyToMany()` method. Provide a common `relationName` on both models to join them into a many-to-many relationship. 
+
+```typescript
+const schema = a.schema({
+  Post: a.model({
+    title: a.string(),
+    content: a.string(),
+    tags: a.manyToMany('Tag', { relationName: 'PostTags'})
+  }),
+  Tag: a.model({
+    name: a.string(),
+    posts: a.manyToMany('Post', { relationName: 'PostTags'})
+  }),
+});
+```
+
+### Create multiple relationships between two models
+
+```typescript
+const schema = a.schema({
+  Employee: a.model({
+    firstName: a.string(),
+    lastName: a.string(),
+  }),
+  Organization: a.model({
+    name: a.string(),
+    individualContributors: a.hasMany('Employee'),
+    managers: a.hasMany('Employee'),
+  }),
+});
+
+const client = generateClient<Schema>();
+
+const { data: org } = await client.models.Organization.get('AE6191BF-FEB4-4D8B-867D-898A7DA33101');
+
+const { data: ics } = await org.individualContributors();
+const { data: managers } = await org.managers();
+```
+
+### Relationships to a model with a composite identifier
+
+TK: stored in a single field with a separator? e.g. `manger.branch = 'TENANT123>Downtown'`
+
+```typescript
+const schema = a.schema({
+  StoreBranch: a.model({
+    tenantId: a.string(),
+    name: a.string(),
+    country: a.string(),
+    state: a.string(),
+    city: a.string(),
+    zipCode: a.string(),
+    streetAddress: a.string(),
+  }).identifier(['tenantId', 'name'])
+});
+```
+
+## Retrieving related models
+
+```typescript
+const schema = a.schema({
+  Project: a.model({
+    name: a.string(),
+    team: a.hasOne('Team'),
+  }),
+  Team: a.model({
+    name: a.string(),
+  }),
+});
+
+const client = generateClient<Schema>();
+
+const { data: project } = await client.models.Project.get('1234');
+project.name;
+(await project.team).name // project.team is typed as `Lazy<Team>` and can be awaited
+
+const projectWithTeam = await client.models.Project.get('1234', ['team.*']);
+project.team.name // All team attributes were requested, so no need to await, it was eagerly loaded
+```
+
+## Manipulating relationships
+
+NO CASCADING BEHAVIOR
+- Save in the right order (e.g. parent before children)
+- What about bi-directionality then? chicken and egg situation
+
+```typescript
+client.Models.Project
+```
+
+## Conclusion
+
+You have now reviewed the major concepts for working with identifiers and modeling relationships. 
+
+### Next steps
+
+Continue to customize your data. We recommend the following next steps:
+
```

Comment:
@renebrandel added some links but calling out in case you want to change where they go for next steps.